File: Keeper.client/src/models/Keep.js
export class Keep {
  constructor(data) {
    this.id = data.id
    this.creatorId = data.creatorId
    this.creator = data.creator
    this.name = data.name
    this.description = data.description
    this.img = data.img
    this.views = data.views
    this.kept = data.kept
    this.tags = data.tags
    this.vaultKeepId = data.vaultKeepId
  }
}

File: Keeper/Models/Keep.cs
namespace Keeper.Models;

public class Keep : DbItem<int>
{
  public string CreatorId { get; set; }
  public Profile Creator { get; set; }
  public string Name { get; set; }
  public string Description { get; set; }
  public string Img { get; set; }
  public int Views { get; set; }
  public int Kept { get; set; }
}
public class KeptKeeps : Keep
{
  public int vaultKeepId { get; set; }
}